src/site/_includes/partials/translation-disclaimer.njk

```diff
@@ -8,7 +8,7 @@
     API.
     {% elif translation === 'manual' %}
     This article was translated from
-    [the original post](/en{{ page.data.canonicalUrl }}).
+    <a href="/en{{ canonicalUrl }}" target="_blank">the original post</a>.
```

Comment:
@samthor could we add a data attribute to links to signal to the router to ignore them? My concern with relying on target="_blank" here is that someone might one day remove it and not realize it has implications with the server.

Reply:
This comment made me realise that we do skip absolute urls in the router, so I changed an internal /en/ link to an absolute one.

Reply:
ah clever!

Would you mind also adding a comment to njk to explain why youre doing the full url